CMakeLists.txt

```diff
@@ -112,30 +116,35 @@ catkin_package(
 ## Your package locations should be listed before other locations
 # include_directories(include)
 include_directories(
-  ${catkin_INCLUDE_DIRS}
+  include ${catkin_INCLUDE_DIRS}
 )
 
 ## Declare a C++ library
-# add_library(iarc7_safety
-#   src/${PROJECT_NAME}/iarc7_safety.cpp
-# )
+add_library(iarc7_safety_safety_client
+  src/${PROJECT_NAME}/SafetyClient.cpp
+)
 
 ## Add cmake target dependencies of the library
 ## as an example, code may need to be generated before libraries
 ## either from message generation or dynamic reconfigure
-# add_dependencies(iarc7_safety ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+add_dependencies(iarc7_safety_safety_client ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+
+## Specify libraries to link a library or executable target against
+target_link_libraries(iarc7_safety_safety_client
+   ${catkin_LIBRARIES}
+ )
 
 ## Declare a C++ executable
-# add_executable(iarc7_safety_node src/iarc7_safety_node.cpp)
+add_executable(iarc7_safety src/SafetyMonitor.cpp)
```

Comment:
Should we be naming our libraries "package name" and the main node "package name node"?